src/librbd/io/ImageDispatchSpec.h

```diff
@@ -128,6 +128,7 @@ class ImageDispatchSpec {
   bool is_write_op() const;
 
   void start_op();
+  void finish_op();
```

Comment:
Perhaps a different name? The op isn't being finished, it's being deferred or paused.